formatTest/unit_tests/expected_output/modules.re

```diff
@@ -445,9 +447,8 @@ let module Id (X: Type) => X;
 let module Compose
            (F: Type => Type)
            (G: Type => Type)
-           (X: Type) => F (
-  G X
-);
+           (X: Type) =>
+  F (G X);
```

Comment:
If function names and their args are in one tuple, I think that functor names and their arguments should be too.